packages/ra-ui-materialui/src/layout/AppBar.tsx

```diff
@@ -61,7 +62,7 @@ const AppBar = ({
 }) => {
     const classes = useStyles({ classes: classesOverride });
     const dispatch = useDispatch();
-    const isXSmall = useMediaQuery(theme => theme.breakpoints.down('xs'));
+    const isXSmall = useMediaQuery<any>(theme => theme.breakpoints.down('xs'));
```

Comment:
Why are you using `any` here ?

```suggestion
    const isXSmall = useMediaQuery<Theme>(theme => theme.breakpoints.down('xs'));
```

Reply:
Don't forget to import the Theme type